example/README.md

```diff
@@ -0,0 +1,16 @@
+# How to test working_dir feature in a remote server (Virtual Machine)
+
+We're going to test the working_dir hapistrano feature by deploying this project
+to a local server (Virtual machine). To do this we need to:
+We're assuming that you have this repo locally.
+
+1. Install virtualbox - link here
```

Comment:
@felixminom maybe we could add a Vagrantfile to the repo in order to make it easier to spin up a VM with the required tools.